react-toggle-switch/my-app/src/ToggleSwitch.tsx

```diff
@@ -0,0 +1,34 @@
+import './ToggleSwitch.css';
+import { useState } from 'react';
+
+export function ToggleSwitch() {
+  const [isOn, setSwitch] = useState(false);
+  let container: string = '';
+  let active: string = '';
+  let label: string = '';
+
+  if (isOn === true) {
+    container = 'container-on';
+    active = 'active';
+    label = 'ON';
+  } else if (isOn === false) {
```

Comment:
Since `isOn` is a boolean, you know its value can only be `true` or `false`, so you don't need to check if it is false here -- it always will be. Instead, just use `} else {`.